specs/capri/README.md

```diff
@@ -0,0 +1 @@
+This folder contains design specs for the Capri release.
```

Comment:
There seems to be a rebase issue.  This should not be shown as a commit from you.